bmq-sdk/src/test/java/com/bloomberg/bmq/it/BrokerSessionIT.java

```diff
@@ -2663,19 +2664,23 @@ public void closeQueuePreconditionFailures() {
      *   <li>stop and linger broker session and stop server
      * </ul>
      */
-    @Test
+    // Temporarily disable this test
```

Comment:
Does it make sense to increase the timeout even more to not disable the tests?

Reply:
Already tried, but it's more complicated. It requires further investigation where exactly it fails, why, how it could be prevented.
Basically the fix should start with logs enhancing for these tests because it's not obvious where the problem started.

Reply:
I think for slow hosts the operation reordering in multithreaded env might happen for these tests. Meaning, that the sequence of expected actions breaks in unusual way, and timeouts for operation execution doesn't even matter.

Reply:
And we see timeout error during final close not because we don't have enough time to exec operation, but because we encountered a failed state on previous steps 😬

Reply:
Got it.
I'm okay to postpone the investigation but suggest to create corresponding issues with necessary information